mixin/thanos/alerts/receive.libsonnet

```diff
@@ -93,6 +93,17 @@
               severity: 'warning',
             },
           },
+          {
+            alert: 'ThanosReceiveNoUpload',
```

Comment:
worth to mention this is only ok for receive with upload option on.... :thinking:

Reply:
are you suggesting a comment?

Reply:
or conditionally add the rule if uploads are enabled?

Reply:
Not want to make things complicated but if we only initialize the shipper for the receive when upload is enabled and check the existence of the metric for the alert, we could achieve this. 
However I think we should defer this until this creates an issue. We can just add a comment.
WDYT?

Reply:
hah you're right, this should already not fire because we only create the shipper and register metrics if enabled